docs/dotnet.rst

```diff
@@ -0,0 +1,132 @@
+   `Flatpak - Quick Setup <https://flatpak.org/setup/>`__
+
+2. Install Flatpak Builder as both a flatpak package and a distribution package (apt / dnf)
+
+.. code-block:: shell
+
+  flatpak install org.flatpak.Builder
+  sudo apt install flatpak-builder
+
+
+Creating the Flatpak
+^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+  Here is a brief description of the placeholders in the below example:
+
+  - ``<app-id>``: The name of your Flatpak, see `"Conventions - Application IDs" section <https://docs.flatpak.org/en/latest/conventions.html#application-ids>`__.
+  - ``<app-name>``: The name of the root folder of your app repository
+  - ``<project-name>``: The name of your ``.csproj`` file
+  - ``<git-server-url>``: The URL to your git server (e.g. ``https://github.com/``, ``https://gitlab.com``)
+  - ``<user-name>``: The username you use in your Git server (GitHub, GitLab, Bitbucket)
+  - ``<release-number>``: The numbered release to build from, in the form of a Git `Tag <https://git-scm.com/book/en/v2/Git-Basics-Tagging>`__.
+
+3.  Create a new folder somewhere different from your existing project
+
+4.  Create a YAML file titled ``<app-id>.yaml`` with the following example template, replacing the placeholders with the appropriate information. (Note: If your project file lives in a subfolder, be sure to include that in the build paths in this file and subsequent commands as well.): 
+
+.. code-block:: yaml
+
+  app-id: <app-id>
+  runtime: org.freedesktop.Platform
+  runtime-version: '23.08'
+  sdk: org.freedesktop.Sdk
+  sdk-extensions:
+    - org.freedesktop.Sdk.Extension.dotnet8
+  build-options:
+    prepend-path: "/usr/lib/sdk/dotnet8/bin"
+    append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+    prepend-pkg-config-path: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig"
+
+  command: <project-name>
+
+  finish-args:
+    - --device=dri
+    # TODO: Replace this with wayland and fallback-x11 once Wayland support
+    #       becomes available:
+    #       https://github.com/AvaloniaUI/Avalonia/pull/8003
+    - --socket=x11
+    - --share=ipc
+    - --env=DOTNET_ROOT=/app/lib/dotnet
+
+  modules:
+    - name: dotnet
+      buildsystem: simple
+      build-commands:
+      - /usr/lib/sdk/dotnet8/bin/install.sh
+
+    - name: <app-id>
+      buildsystem: simple
+      sources:
+        - type: git
+          url: <git-server-url>/<user-name>/<project-name>.git
+          tag: <release-number>
+        - ./nuget-sources.json
+      build-commands:
+        - dotnet publish <project-name>.csproj -c Release --no-self-contained --source ./nuget-sources
+        - mkdir -p ${FLATPAK_DEST}/bin
+        - cp -r /bin/Release/net8.0/publish/* ${FLATPAK_DEST}/bin
+
+.. note::
+
+    For providing access to other things such as the network or
+    filesystem, see the `“Sandbox Permissions” section <https://docs.flatpak.org/en/latest/sandbox-permissions.html>`__
```

Comment:
Use :doc:sandbox-permissions, surround sandbox-permissions with backticks on front and back.